samples/instanceadmin/instanceadmin.py

```diff
@@ -1,259 +1,270 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstrates how to connect to Cloud Bigtable and run some basic operations.
+#     http://www.apache.org/licenses/LICENSE-2.0
+Prerequisites:
+- Create a Cloud Bigtable project.
+  https://cloud.google.com/bigtable/docs/
+- Set your Google Application Default Credentials.
+  https://developers.google.com/identity/protocols/application-default-credentials
+
+Operations performed:
+- Create a Cloud Bigtable Instance.
+- List Instance for a Cloud Bigtable.
+- Delete a Cloud Bigtable Instance.
+- Create a Cloud Bigtable Cluster.
+- List Cloud Bigtable Clusters.
+- Delete a Cloud Bigtable Cluster.
+"""
+
+import argparse
+
+from google.cloud import bigtable
+from google.cloud.bigtable import enums
+
+
+def run_instance_operations(project_id, instance_id, cluster_id):
+    """Check Instance exists.
+        Creates a Production instance with default Cluster.
+        List instances in a project.
+        List clusters in an instance.
+
+    :type project_id: str
+    :param project_id: Project id of the client.
+
+    :type instance_id: str
+    :param instance_id: Instance of the client.
+    """
+    client = bigtable.Client(project=project_id, admin=True)
+    location_id = "us-central1-f"
+    serve_nodes = 1
+    storage_type = enums.StorageType.SSD
+    production = enums.Instance.Type.PRODUCTION
+    labels = {"prod-label": "prod-label"}
+    instance = client.instance(instance_id, instance_type=production, labels=labels)
+
+    # [START bigtable_check_instance_exists]
+    if not instance.exists():
+        print("Instance {} does not exist.".format(instance_id))
+    else:
+        print("Instance {} already exists.".format(instance_id))
+    # [END bigtable_check_instance_exists]
+
+    # [START bigtable_create_prod_instance]
+    cluster = instance.cluster(
+        cluster_id,
+        location_id=location_id,
+        serve_nodes=serve_nodes,
+        default_storage_type=storage_type,
+    )
+    if not instance.exists():
+        print("\nCreating an instance")
+        # Create instance with given options
+        instance.create(clusters=[cluster])
+        print("\nCreated instance: {}".format(instance_id))
+    # [END bigtable_create_prod_instance]
+
+    # [START bigtable_list_instances]
+    print("\nListing instances:")
+    for instance_local in client.list_instances()[0]:
+        print(instance_local.instance_id)
+    # [END bigtable_list_instances]
+
+    # [START bigtable_get_instance]
+    print(
+        "\nName of instance: {}\nLabels: {}".format(
+            instance.display_name, instance.labels
+        )
+    )
+    # [END bigtable_get_instance]
+
+    # [START bigtable_get_clusters]
+    print("\nListing clusters...")
+    for cluster in instance.list_clusters()[0]:
+        print(cluster.cluster_id)
+    # [END bigtable_get_clusters]
+
+
+def create_dev_instance(project_id, instance_id, cluster_id):
```

Comment:
We don't need this anymore. From the docs:

"Note: You are no longer able to create new development instances, because production instances of only one or two clusters are now generally available."

Reply:
a8f33ac